client/reducers/error.js

```diff
@@ -0,0 +1,24 @@
+import axios from 'axios';
+
```

Comment:
let's write some comments as to how this Error reducer should be used, and that it is only in relation to the signup error not any other errors, as of right now

Reply:
done